Orleans.Providers.MongoDB/StorageProviders/Serializers/IGrainStateSerializer.cs

```diff
@@ -1,11 +1,11 @@
-﻿using Newtonsoft.Json.Linq;
+﻿using MongoDB.Bson;
 
-namespace Orleans.Providers.MongoDB.StorageProviders
+namespace Orleans.Providers.MongoDB.StorageProviders.Serializers
 {
     public interface IGrainStateSerializer
```

Comment:
Shall we not document every public type and method?

Reply:
Updated.